libc/src/__support/FPUtil/FPBits.h

```diff
@@ -39,41 +39,41 @@ enum class FPEncoding {
   X86_ExtendedPrecision,
 };
 
-template <FPType> struct FPBaseProperties {};
+template <FPType> struct FPBaseAttr {};
 
-template <> struct FPBaseProperties<FPType::IEEE754_Binary16> {
+template <> struct FPBaseAttr<FPType::IEEE754_Binary16> {
```

Comment:
Would it make sense to derive `TOTAL_LEN` from `SIG_LEN`, `EXP_LEN` and `ENCODING` (in a derived class)?